addons/main/functions/fnc_addActionsToUnit.sqf

```diff
@@ -29,10 +29,6 @@ private _arr = [_unit, localize "str_heal", "\a3\ui_f\data\IGUI\Cfg\holdactions\
     // codeCompleted
     params ["_target", "_caller"];
     call FUNC(deleteProgressBar);
-    private _ret = [_caller] call FUNC(hasHealItems);
-    if (_ret isEqualTo 1) then {
-        _caller removeItem "FirstAidKit";
-    };
```

Comment:
Unsure about this change, this will make reviving people not remove a FAK if not a medic.

Reply:
I don't think so? This is the code for reviving:

```sqf
{
    _unit setDamage 0;
    [_unit, _healer] call FUNC(handleHealEh);
};
```

It calls the heal handler no matter what after being revived, which now consumes the medkit.

Reply:
Tested all scenarios:
```
Revive as non-medic: 1 FAK consumed
Heal other as non-medic: 1 FAK consumed
Heal self as non-medic: 1 FAK consumed
Revive as medic without Medikit: 1 FAK consumed
Heal self as medic without Medikit: 2 FAKs consumed
Heal other as medic without Medikit: 2 FAKs consumed
Revive as medic: nothing consumed
Heal self as medic: nothing consumed
Heal other as medic: nothing consumed
```
Unsure why the medkit-less medic healing behavior is aberrant.